Export jose-util helpers

aeneasr opened the pull request:
The utils for key generations are very helpful in other projects as well. Exporting them seems like a good idea without any downsides as the code is properly tested.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/square/go-jose?pullRequest=300) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/square/go-jose?pullRequest=300) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/square/go-jose?pullRequest=300) it.</sub>

mbyczkowski commented on jose-util/go.sum:
```diff
@@ -6,6 +6,8 @@ github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
+github.com/square/go-jose v1.1.2 h1:YCLGvrhulDcHE8AyoqKUKMaY7fQL9tg0c3Uh4QWvQzQ=
```
this seems like an accidental import somewhere?

aeneasr replied:
I'll run go mod tidy again. I didn't change anything in the code base except move the functions around.

aeneasr replied:
Nevermind, you're correct - an incorrect import (without v3) was used somewhere. Nice catch!

aeneasr commented:
Thank you for the review, I addressed your feedback. Given that we use `cipher`, `cryptosigner`, `jwt` I assumed that `generator.` would be prefered to `generators.`. It's now `generator.NewSigningKey()` - looks pretty sweet to me!

mbyczkowski commented:
Sorry @aeneasr this got lost in my inbox 😅

aeneasr commented:
All good, thanks for the merge! :)
